### ISSUE TITLE
Cannot find a types declaration

### ISSUE THREAD

**Issue by xantinium** [Sphinxxxx/vanilla-picker#51]
I ran into a problem that when using the `moduleResolution` option with the value `"Bundle"` (in `tsconfig.json`), it is not possible to find the types for the package.
![Screenshot from 2024-03-22 12-55-28](https://github.com/Sphinxxxx/vanilla-picker/assets/115897451/b451773b-de11-48c2-8ffa-968ec1a85a63)

I found a solution: you need to make edits to `package.json`.
```
"exports": {
    ".": {
      "require": "./dist/vanilla-picker.js",
      "import": "./dist/vanilla-picker.mjs",
      "default": "./dist/vanilla-picker.mjs",
      "types": "./src/index.d.ts"   <---   ADD THIS LINE
    },
    "./csp": {
      "require": "./dist/vanilla-picker.csp.js",
      "import": "./dist/vanilla-picker.csp.mjs",
      "default": "./dist/vanilla-picker.csp.mjs"
    },
    "./dist/": "./dist/",
    "./dist/*": "./dist/*"
  },
```

Can u help me please? I know, it's minor, but very important in my case

**Comment by Sphinxxxx** [comment]
Published a new version, `2.12.3`.
Does that work as intended?

*Note to self: https://stackoverflow.com/questions/58990498/package-json-exports-field-not-working-with-typescript*

**Comment by Sphinxxxx** [comment]
Closing this. Let me know if it's still a problem..
